docs/4.Integrations/Terraform Scanning.md

```diff
@@ -20,8 +20,29 @@ terraform show -json tfplan.binary > tfplan.json
 checkov -f tfplan.json
 ```
 
+Note: The Terraform show output file `tf.json` will be a single line. For that reason Checkov will report all findings as line number 0.
+If you have installed jq, you can convert a JSON file into multiple lines making it easier to read the scan result.
+
+```json
+terraform plan --out tfplan.binary
```

Comment:
why arer we adding this example? (lines 26 to 30) - it is already in line 16.

Reply:
only because I felt a more complete example makes more sense, but it's not the way the rest of the docs do it, so I'll remove this